.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
```

Comment:
Description needs to also include `BREAKING CHANGE:` if it's not backwards compatible (i.e. major bump)